tests/Feature/UserTest.php

```diff
@@ -14,35 +14,57 @@ class UserTest extends TestCase
     /** @test */
     public function create_user()
     {
-        $request = [
-            'username' => 'Luke',
-            'location' => '51.498134,-0.201755'
-        ];
+        $user = User::factory()->make();
```

Comment:
De Unittesten moest ik aanpassen, omdat ik de database had gewijzigd. Verder heb ik ook de Unittesten een klein beetje herschreven, zodat het een standaard structuur heeft, zodat het makkelijk wordt om andere Unittesten te schrijven.

Met `User::factory()->make();` laat je één user generen. Met `$user = User::factory()->create();` maakt die een gebruiker aan en slaat die op in de database.